Test/git-issues/git-issue-401a.dfy.expect

```diff
@@ -0,0 +1 @@
+*** ProverException: Cannot find prover specified with z3exe: /Users/davidcok/yucca/dafny/Test/git-issues/../../Binaries/z3/binz/z3
```

Comment:
Can we do something to make this path come out without a specific user's directory? Would just making it a relative path (`Output/binz`) do the trick?